src/Clients/Event/Config.php

```diff
@@ -0,0 +1,19 @@
+<?php declare(strict_types=1);
+
+namespace StrictPhp\HttpClients\Clients\Event;
+
+use StrictPhp\HttpClients\Contracts\ConfigContract;
+use StrictPhp\HttpClients\Entities\AbstractConfig;
+
+final class Config extends AbstractConfig
```

Comment:
Za mě není ideální, že se jmenujou všechny configy Config.

Bude se to blbě hledat a budu muset dělat aliasy - za mě bad DX.

Byl bych radši Aby to bylo StoreConfig, SleepConfig. Takto jsem hned schopen najít ten config a použít.

Reply:
upraveno